restapi_test.go

```diff
@@ -250,6 +251,27 @@ func TestChannelMessageSend2(t *testing.T) {
 	}
 }
 
+// TestChannelMessageSendComplex_files tests sending files via
+// ChannelMessageSendComplex() function. This should not return an error.
+func TestChannelMessageSendComplex_files(t * testing.T) {
```

Comment:
Please run `gofmt` on your files before committing; this is part of what the Travis build is complaining about.